.gitignore

```diff
@@ -0,0 +1,7 @@
+*.gcompcache
+*.lvprojectcache
+*.niaddoncache
+*.lvcodedb
+*.aliases
+*.lvlps
+*.gcompcache-journal
```

Comment:
I'm leaving `Builds` in due to the fact we don't know how to do LV builds on CI infrastructure (maybe a good hackathon topic ;-). 

Git wasn't ignoring the .cache directory. In fact I commit all those files while troubleshooting another issue with Jaidev. See https://github.com/prestwick/niweek-hero-2017/tree/mod-gitignore